tests/test_network_extensions.py

```diff
@@ -220,3 +220,22 @@ def test_load_detail():
 
     n.remove_extensions(extension_name, [element_id])
     assert n.get_extensions(extension_name).empty
+
+def test_measurements():
+    n = pn.create_four_substations_node_breaker_network()
+    extension_name = 'measurements'
+    element_id = 'LD1'
+    extensions = n.get_extensions(extension_name)
+    assert extensions.empty
+
+    n.create_extensions(extension_name, id='measurement1', element_id=element_id, type='CURRENT', value=100, standard_deviation=2, valid=True)
```

Comment:
we can maybe add a test with multiple measurements on the same element, for example it's typical to have both P and Q measurements for a branch or an injection